Dockerfile

```diff
@@ -14,5 +14,4 @@ FROM debian:buster-slim
 WORKDIR /app
 COPY --from=0 /app/out/raccoon ./raccoon
 COPY . .
-EXPOSE 8080
```

Comment:
`EXPOSE` doesn't actually [expose the port](https://docs.docker.com/engine/reference/builder/#expose). Since Raccoon port is configurable, it's better to remove it.